src/DataTableTypeWithOptionsInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * Symfony DataTables Bundle
+ * (c) Omines Internetbureau B.V. - https://omines.nl/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Omines\DataTablesBundle;
+
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+
+interface DataTableTypeWithOptionsInterface extends DataTableTypeInterface
```

Comment:
Just a naming suggestion : OptionsAwareDatableTypeInterface ?

Reply:
Is it that obvious that I struggled to find a good name?...
OptionsAwareDatableTypeInterface sounds good 👍